api/server/src/main/java/jakarta/websocket/server/ServerEndpointConfig.java

```diff
@@ -145,7 +145,8 @@ public String getNegotiatedSubprotocol(List<String> supported, List<String> requ
          * The default platform implementation of this method returns a list containing all of the requested extensions
          * passed to this method that it supports, using the order in the requested extensions, the empty list if none.
          *
-         * @param installed the installed extensions on the implementation.
+         * @param installed the union of the extensions defined for the endpoint and
+         *            {@link ServerContainer#getInstalledExtensions()}.
```

Comment:
What does `the extensions defined for the endpoint` refer to?

Is this referring to the `ServerEndpointConfig.getExtensions` method? if so I believe the javadoc on the `ServerEndpointConfig.getExtensions` needs to be clarified because it is not clear what the intended purpose is.